go/kbfs/libfuse/mounter.go

```diff
@@ -32,6 +32,14 @@ type mounter struct {
 // fuseMount tries to mount the mountpoint.
 // On a force mount then unmount, re-mount if unsuccessful
 func (m *mounter) Mount() (err error) {
+	defer func() {
+		if err != nil {
+			msg := fmt.Sprintf("KBFS failed to FUSE mount at %s: %s", m.options.MountPoint, err)
+			fmt.Println(msg)
+			m.log.Warning(msg)
```

Comment:
![2019-01-16-164006_2214x1039_scrot](https://user-images.githubusercontent.com/28712558/51280343-65d4eb00-19ad-11e9-9b53-9e8e3bf1459e.png)

I closed kbfs, put a file in my mountdir, and started it back up. Now the mount error goes to both stdout and the log. I'm not sure if this is hacky or not but it works